src/turtle.rs

```diff
@@ -1156,4 +1158,22 @@ mod tests {
         assert_eq!(turtle.position()[1].round(), 100.0);
         assert_eq!(turtle.heading(), 51.0);
     }
+
+    #[test]
+    fn turn_towards() {        
+        let mut turtle = Turtle::new();
+        
+        // Turn from each cardinal direction to each cardinal direction
+        for n in 0..16 as u32 {
+            let original_angle = radians::TWO_PI * n as f64 / 16.0;
+            turtle.turn_towards([original_angle.cos(), original_angle.sin()]);
+            assert_eq!(turtle.heading().ceil(), original_angle.to_degrees().ceil());
```

Comment:
Please use `round()` instead of `ceil()` because we want to make sure that rounding differences are caught by these tests.